Source/Actors/Player.cs

```diff
@@ -256,6 +256,7 @@ public bool IsAbleToPause
         GetCameraTarget(out var orig, out var target, out _);
         World.Camera.LookAt = target;
         World.Camera.Position = orig;
+        World.Camera.FarPlane = Save.Instance.SimplifiedGraphics ? 8000 : 800;
```

Comment:
this was a fix for the Cameras Farplane being set to 100, which is weird, since the code isn't supposed to do that